database-plugins/src/main/java/io/cdap/plugin/DBRecord.java

```diff
@@ -196,7 +196,7 @@ private void writeToDataOut(DataOutput out, Schema.Field field) throws IOExcepti
         out.write((byte[]) fieldValue);
         break;
       default:
-        throw new IOException(String.format("Unsupported datatype: %s with value: %s.", fieldType, fieldValue));
+        throw new IOException(String.format("Unsupported datatype: %s with value: %s and name: %s.", fieldType, fieldValue, field.getName()));
```

Comment:
I think we should fix the grammar in this sentence. How about - `Column %s with value %s has an unsupported datatype %s`.